indico/htdocs/js/indico/Core/Widgets/UserList.js

```diff
@@ -509,19 +503,17 @@ type("ListOfUsersManagerForForm", ["ListOfUsersManager"], {
                                        'phone': any(newData.get('phone'), ''),
                                        'fax': any(newData.get('fax'), '')
                                       };
-                    if (newUserPopup.parameterManager.check()) {
-                        if (!self._isAlreadyInList(newUserData['email'])) {
-                            newUserPopup.close();
-                            self.usersList.append(newUserData);
-                            self._drawUserList();
-                        } else {
-                            var popup = new AlertPopup($T('Add ')+self.userCaption,
-                                    $T('The email address (') + newUserData['email'] +
-                                    $T(') is already used by another participant. Please modify email field value.'));
-                            popup.open();
-                        }
+                    if (!self._isAlreadyInList(newUserData['email'])) {
+                        newUserPopup.close();
+                        self.usersList.append(newUserData);
+                        self._drawUserList();
+                    } else {
+                        var popup = new AlertPopup($T('Add ')+self.userCaption,
+                                $T('The email address (') + newUserData['email'] +
```

Comment:
`$T.gettext(...)` and `newUserData.email` - but use `.format()` instead of this cancerous concatenation of translatable strings.